colour/models/rgb/cylindrical.py

```diff
@@ -336,3 +343,184 @@ def H_to_RGB(vi, vj, vH):
     RGB = tstack([R, G, B])
 
     return from_range_1(RGB)
+
+
+def RGB_to_HCL(RGB, gamma=3, Y_0=100):
+    """
+    Converts from *RGB* colourspace to *HCL* colourspace.
+
+    Parameters
+    ----------
+    RGB : array_like
+        *RGB* colourspace array.
+    gamma : double, optional
```

Comment:
Should be `numeric, optional` and we need parameter description likewise for `Y_0`.

Reply:
Oh yes, somehow missed the Y_0 parameter,